restcomm/restcomm.commons/src/main/java/org/restcomm/connect/commons/util/DigestAuthentication.java

```diff
@@ -80,11 +102,20 @@ private static String H(final String data) {
     }
 
     private static String KD(final String secret, final String data) {
-        return H(secret + ":" + data);
+        return KD(secret, data, "MD5");
```

Comment:
@abdulazizali77 why do we hardcode to MD5 instead of using client-algorithm from the settings ?